az-digital/az_quickstart
# Closes #1404: Push sidebar down and fix horizontal scroll problem.

## Description


## Related Issue




#1404 

## How to test?
Any page with both a full-width element link a text on media or text on background paragraph, AND one or two sidebars should be affected by this PR.

In any browser you deem supportable, add a page with a full-width paragraph AND other paragraphs.  Then add a block or two to either side of the page.


The sidebar should appear beneath the last full-width element on the page.
The rows of all of the full width paragraphs should be the width of the page container.

## Types of changes

**Arizona Quickstart** (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

**Drupal core**
- Patch release changes
- Minor release changes

**Drupal contrib projects**
- Patch release changes
- Minor release changes

## Checklist:
